src/Mirage-Carousel/MICarousel.class.st

```diff
@@ -26,8 +26,8 @@ MICarousel class >> activate: aBoolean [
 
 { #category : #'initialize-release' }
 MICarousel class >> initialize [
-	super initialize.
```

Comment:
Why removing the call to super here?

Reply:
While reading your code, I saw a code critic rule discouraging sending `#initialize` to `super` on the class-side since some class of the top of the hierarchy should not be reinitialized.

I also browsed the Morph hierarchy and I didn't find a single `super initialize` collaboration on the class initialization.

Reply:
Ah yeah, you're right I didn't realised it was on class side.